### PR TITLE
zh-CN: update learn/server-side/express_nodejs/forms/create_book_form

### PR DESCRIPTION
**Pull request by PassionPenguin**

### CODE REVIEW

**Comment by PassionPenguin** [comment]
PTAL @yin1999

File: files/zh-cn/learn/server-side/express_nodejs/forms/create_book_form/index.md [hunk 1]
```diff
@@ -220,16 +220,16 @@ block content
...
-主要区别在于，我们如何实现选择类型字段：作者`Author`和种类`Genre`。
+主要区别在于，我们如何实现选择类型字段：作者 `Author` 和种类 `Genre`。
 
-- 种类集合显示为复选框，使用我们在控制器中设置的检查值`checked`，来确定是否应该选中该框。
-- 作者集合显示为单选下拉列表。在这种情况下，我们通过比较当前作者选项的 id 与用户先前输入的值（作为`book`变量传入），来确定要显示的作者。这在上面突出显示！
+- 种类集合显示为复选框，使用我们在控制器中设置的检查值 `checked`，来确定是否应该选中该框。
+- 作者集合显示为单选下拉列表。在这种情况下，我们通过比较当前作者选项的 id 与用户先前输入的值（作为 `book` 变量传入），来确定要显示的作者。这在上面突出显示！
 
-  > **备注：** 如果提交的表单中存在错误，那么，当要重新呈现表单时，新的书本作者仅使用字符串（作者列表中选中选项的值）进行标识。相比之下，现有的书本作者的`_id`属性不是字符串。因此，要比较新的和现有的，我们必须将每个现有书本作者的`_id`，强制转换为字符串，如上所示。
+  > **注意：** 如果提交的表单中存在错误，那么，当要重新呈现表单时，新的书本作者仅使用字符串（作者列表中选中选项的值）进行标识。相比之下，现有的书本作者的 `_id` 属性不是字符串。因此，要比较新的和现有的，我们必须将每个现有书本作者的 `_id`，强制转换为字符串，如上所示。
```
Comment (yin1999) [inline files/zh-cn/learn/server-side/express_nodejs/forms/create_book_form/index.md:228]: https://github.com/mdn/translated-content/pull/19515#discussion_r1564634645

**Review by yin1999 — CHANGES_REQUESTED** [review]

File: files/zh-cn/learn/server-side/express_nodejs/forms/create_book_form/index.md (hunk 1 above)
Comment (PassionPenguin) [reply files/zh-cn/learn/server-side/express_nodejs/forms/create_book_form/index.md:228]:
```suggestion
  > **备注：** 如果提交的表单中存在错误，那么，当要重新呈现表单时，新的书本作者仅使用字符串（作者列表中选中选项的值）进行标识。相比之下，现有的书本作者的 `_id` 属性不是字符串。因此，要比较新的和现有的，我们必须将每个现有书本作者的 `_id`，强制转换为字符串，如上所示。
```

File: files/zh-cn/learn/server-side/express_nodejs/forms/create_book_form/index.md [hunk 2]
```diff
@@ -220,16 +220,16 @@ block content
...
+- 种类集合显示为复选框，使用我们在控制器中设置的检查值 `checked`，来确定是否应该选中该框。
+- 作者集合显示为单选下拉列表。在这种情况下，我们通过比较当前作者选项的 id 与用户先前输入的值（作为 `book` 变量传入），来确定要显示的作者。这在上面突出显示！
 
-  > **备注：** 如果提交的表单中存在错误，那么，当要重新呈现表单时，新的书本作者仅使用字符串（作者列表中选中选项的值）进行标识。相比之下，现有的书本作者的`_id`属性不是字符串。因此，要比较新的和现有的，我们必须将每个现有书本作者的`_id`，强制转换为字符串，如上所示。
+  > **备注：** 如果提交的表单中存在错误，那么，当要重新呈现表单时，新的书本作者仅使用字符串（作者列表中选中选项的值）进行标识。相比之下，现有的书本作者的 `_id` 属性不是字符串。因此，要比较新的和现有的，我们必须将每个现有书本作者的 `_id`，强制转换为字符串，如上所示。
 
 ## 它看起來像是？
 
-运行应用程序，将浏览器打开到 `http://localhost:3000/`，然后选择 Create new book 链接。如果一切设置正确，你的网站应该类似于以下屏幕截图。提交有效的图书后，应将其保存，然后你将进入图书详细信息页面。
+运行应用程序，将浏览器打开到 `http://localhost:3000`，然后选择 Create new book 链接。如果一切设置正确，你的网站应该类似于以下屏幕截图。提交有效的图书后，应将其保存，然后你将进入图书详细信息页面。
```
Comment (yin1999) [inline files/zh-cn/learn/server-side/express_nodejs/forms/create_book_form/index.md:232]: Please revert this: https://developer.mozilla.org/en-US/docs/Learn/Server-side/Express_Nodejs/forms/Create_book_form#what_does_it_look_like

Comment (PassionPenguin) [reply files/zh-cn/learn/server-side/express_nodejs/forms/create_book_form/index.md:232]: ahh the link is not broken in GFM.

as i was editing in webstorm, the backslash seems to break the backtick.

File: files/zh-cn/learn/server-side/express_nodejs/forms/create_book_form/index.md [hunk 3]
```diff
@@ -157,13 +157,13 @@ sanitizeBody('*').trim().escape(),
 },
 ```
 
-然后，我们在清理器中使用通配符（\*）来单独验证每个种类数组条目。下面的代码显示了 - 这转换为“清理关键种类`genre`下的每个项目”。
+然后，我们在清理器中使用通配符（\*）来单独验证每个种类数组条目。下面的代码显示了 - 这转换为“清理关键种类 `genre` 下的每个项目”。
```
Comment (yin1999) [inline files/zh-cn/learn/server-side/express_nodejs/forms/create_book_form/index.md:160]:
```suggestion
然后，我们在清理器中使用通配符（\*）来单独验证每个种类数组条目。下面的代码显示了如何操作——这转换为“清理关键种类 `genre` 下的每个项目”。
```

**Review by yin1999 — APPROVED** [review]
Thanks @PassionPenguin

**Comment by j0078910** [comment]
J

**Pull request merged**